### DISCUSSION TITLE
Is it possible to migrate from Sonoff ZBDongle-E to older model ZBDongle-P without re-pairing?

### DISCUSSION THREAD

**Discussion by mdzhigarov** [Koenkk/zigbee2mqtt#17675] [Devices]
I have a ZBDONGLE-E coordinator and I'm not happy with it. Constant problems with unreachable devices and networking issues.
So i ordered the older version - ZBDONGLE-P. I expect a delivery tomorrow.

Is it possible to migrate all my paired devices from Sonoff ZBDongle-E to older model ZBDongle-P without re-pairing?

I saw this in the FAQ https://www.zigbee2mqtt.io/guide/faq/#how-do-i-migrate-from-one-adapter-to-another and it says this is supported for the Zstack adapter but what does that exactly mean? Is the ZBDongle-E covered?

**Comment by Koenkk** [comment]
ZBDongle-E is not covered, re-pairing is necessary since for the ZBDongle-E no backup is supported.

**Reply by mdzhigarov** [reply]
That's unfortunate... Are the IDs of the zigbee devices preserved at least after migration? If not, that'd mean I have to rewrite all my automations and configurations

**Reply by Koenkk** [reply]
Yes ID's are preserved.

**Comment by Hedda** [comment]
> Is it possible to migrate from Sonoff ZBDongle-E to older model ZBDongle-P without re-pairing

See existing discussion and workaround for migrating -> https://github.com/Koenkk/zigbee2mqtt/discussions/16478
